repos/system_upgrade/el7toel8/actors/repositoriesmapping/actor.py

```diff
@@ -15,17 +18,25 @@ class RepositoriesMapping(Actor):
     produces = (RepositoriesMap,)
     tags = (IPUWorkflowTag, ChecksPhaseTag)
 
-    
     def process(self):
+        file_path = '/etc/leapp/files/repomap.csv'
         repos_maps = RepositoriesMap()
 
-        with open(self.get_file_path('repomap.csv')) as f:
+        if not os.path.isfile(file_path):
+            title = 'Repositories data file is invalid (/etc/leapp/files/repomap.csv)'
+            summary = 'Read documentation at: https://access.redhat.com/articles/3664871 for more information ' \
+                'about how to retrieve the files'
+            reporting.report_generic(title=title, summary=summary, severity='high', flags=['inhibitor'])
+            raise StopActorExecutionError(message=title, details={'details': summary})
+
+        with open(file_path) as f:
+            print(f.readlines())
             data = csv.reader(f)
```

Comment:
Would be good to have some check for repomap.csv file content. Maybe check if csv.reader was able to read the file and notify user if some problem happens.